File: fizzbuzz.cpp
#include <string>
#include <iostream>

using namespace std;

int main()
{
    cout << "Please input an integer greater than 1." << endl;
    int n;
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        if (i % 3 == 0)
        {
            if (i % 5 == 0)
            {
                cout << "FizzBuzz" << endl;
            }
            else 
            {
                cout << "Fizz" << endl;
            }
        }
        else if (i % 5 == 0)
        {
            cout << "Buzz" << endl;
        }
        else
        {
            cout << i << endl;
        }
    }
}

File: fizzbuzz.java
import java.util.*;

public Class FizzBuzz
{
    public static void main (String[] args)
    {
        System.out.println("Please input an integer greater than 1.");
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for (int i = 1; i <= n; i++)
        {
            if (i % 3 == 0)
            {
                if (i % 5 == 0)
                {
                    System.out.println("FizzBuzz");
                }
                else 
                {
                    System.out.println("Fizz");
                }
            }
            else if (i % 5 == 0)
            {
                System.out.println("Buzz");
            }
            else
            {
                System.out.println(i);
            }
        }
    }
}
